ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/LedgerDao.scala

```diff
@@ -144,20 +144,13 @@ trait LedgerReadDao extends ReportsHealth {
     *
     * @param deduplicationKey The key used to deduplicate commands
     * @param submittedAt The time when the command was submitted
-    * @param ttl The time until which the command should be deduplicated
+    * @param deduplicateUntil The time until which the command should be deduplicated
     * @return
     */
   def deduplicateCommand(
       deduplicationKey: String,
       submittedAt: Instant,
-      ttl: Instant): Future[Option[CommandDeduplicationEntry]]
-
-  /** Sets the result of a command, so that duplicate submissions can return the original result */
-  def updateCommandResult(
-      deduplicationKey: String,
-      submittedAt: Instant,
-      code: Int,
-      message: Option[String]): Future[Unit]
+      deduplicateUntil: Instant): Future[Option[CommandDeduplicationEntry]]
```

Comment:
I think we can use the type `CommandDeduplicationResult` directly here. No need to add a layer of translation. What do you think?

Reply:
Initially, the ledger DAO tried to only use its own data types, but I see it already depends on `com.daml.ledger.participant.state.index.v2.PackageDetails`